Geometry/ForwardGeometry/src/ZdcHardcodeGeometryLoader.cc

```diff
@@ -34,6 +34,7 @@ ZdcHardcodeGeometryLoader::load(DetId::Detector /*det*/, int subdet)
       fill(HcalZDCDetId::EM  ,hg );
       fill(HcalZDCDetId::LUM ,hg );
       fill(HcalZDCDetId::HAD ,hg );
+//      fill(HcalZDCDetId::RPD ,hg );
```

Comment:
@bsunanda can confirm, but I think this change is made to avoid having to make a geometry for the RPD at this point. Right now we just want to unpack RAW data, so we don't need a geometry. (Previously, it was crashing trying to load a non-existent geometry from the ZDC geometry in the database.) Eventually we might add RPD to the simulation, but that is a separate project for the future.